robowflex_ompl/src/ompl_interface.cpp

```diff
@@ -112,6 +114,8 @@ void OMPL::OMPLInterfacePlanner::refreshContext(const SceneConstPtr &scene,
         return;
     }
 
+    context_->setInterpolation(interpolate_);
+    context_->setHybridize(hybridize_);
```

Comment:
Looks like these functions only got added in melodic, you should add a header blocker around them.

Reply:
Yeah, I was the one who added them actually